app/terms/page.tsx

```
@@ -0,0 +1,220 @@
+          </div>
+        </section>
+
+        {/* Content Section */}
+        <div className="mx-auto max-w-4xl px-4 py-16 sm:px-6 lg:px-8">
+          <div className="rounded-2xl border border-white/70 bg-white/70 p-6 shadow-xl shadow-violet-950/10 backdrop-blur-xl dark:border-white/10 dark:bg-white/[0.04] dark:shadow-black/20 sm:p-8 md:p-10">
+            <div className="prose prose-zinc max-w-none dark:prose-invert prose-headings:text-zinc-950 dark:prose-headings:text-white prose-h1:text-3xl prose-h2:text-2xl prose-h2:font-semibold prose-h2:mt-8 prose-h2:mb-4 prose-p:text-zinc-600 dark:prose-p:text-zinc-300 prose-strong:text-zinc-900 dark:prose-strong:text-white prose-li:text-zinc-600 dark:prose-li:text-zinc-300">
+              
+              <section>
+                <h2>1. Introduction</h2>
+                <p>
+                  Welcome to <strong>LinkID</strong> ("we," "our," or "us"). By accessing or using our platform at{' '}
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&`#34`;`, `&rdquo;`.

(react/no-unescaped-entities)